airflow/config_templates/config.yml

```diff
@@ -271,8 +270,8 @@ core:
       default: "0"
     default_task_retry_delay:
       description: |
-          The number of seconds each task is going to wait by default between retries. Can be overridden at
```

Comment:
Can you please remove the unrelated changes to a separate PR ? I understand this is done by your editor automatically and it's cool to habve things fixed, but there are certain properties of such changes (for example being cherry-pickable) that calls for separating unrelated changes to separate PRs.

Reply:
Can do 👍